packages/sanes-chrome-extension/src/extension/backgroundscript.ts

```diff
@@ -22,6 +22,26 @@ async function handleMessageToBackground(
         console.log('Received message from: ' + sender.tab.id);
       }
       return { msg: 'Hello mate' };
+    case MessageToBackgroundAction.GetPersona: {
+      if (sender.id !== chrome.runtime.id) {
```

Comment:
This condition should also be tested in an e2e test. IF this is out of the scope of this PR, create a ticket and assign it to me.

Reply:
https://github.com/iov-one/ponferrada/issues/145